src/screenshots/crop.js

```diff
@@ -50,7 +47,7 @@ function markSeedToId (markSeed) {
     return id;
 }
 
-function detectClippingArea (srcImage, { markSeed, clientAreaDimensions, cropDimensions, screenshotPath } = {}) {
+function detectClippingArea (srcImage, markSeed, clientAreaDimensions, cropDimensions) {
```

Comment:
Why do you ungroup the parameters?

Reply:
I found it unreasonable to construct an object to pass it as an argument, and then immediately destructure this object. Using parameters in the usual manner are sufficient here.